server/controllers/users/getAllUsersEndpoint.js

```diff
@@ -0,0 +1,4 @@
+export default async (req, res) => {
+    console.log('all users return here')
+    return res.status(200).json({ users: [1, 2] })
```

Comment:
what does this [1,2] return ?

Reply:
it's dummy data. Returns a `user` object with an array of 1 and 2. This is a placeholder for now cuz i was just testing routes to make sure my node server was set up to run them